packages/extension/src/providers/polkadot/networks/interlay/types/interlay-orml-asset.ts

```diff
@@ -0,0 +1,54 @@
+import { ApiPromise } from "@polkadot/api";
+import { OrmlTokensAccountData } from "../../acala/types/acala-orml-asset";
+import { SubmittableExtrinsic } from "@polkadot/api/types";
+import { ISubmittableResult } from "@polkadot/types/types";
+import { SubstrateToken } from "@/providers/polkadot/types/substrate-token";
+import { BaseTokenOptions } from "@/types/base-token";
+
+export type OrmlAssetType =
```

Comment:
For Interlay, should be these token types: https://docs.interlay.io/#/developers/assets?id=token-types

Not sure though why they redefine the token types in two files.